{
    "directory name": "llvm-project/lldb/test/API/python_api/sberror",
    "score.sum": 9.983304870887515e-06,
    "sha.latest": "2238dcc39358353cac21df75c3c3286ab20b8f53",
    "time.latest": 1717966428.3562841,
    "formatted_time": "2024-06-10 02:23:48.356286",
    "filescores": {
        "TestSBError.py": 9.983304870887515e-06
    },
    "subdirscores": {}
}